FileShareAdminClient/FileShareApiAdminClient.cs

```diff
@@ -205,5 +207,20 @@ public async Task RollBackBatchAsync(IBatchHandle batchHandle)
                 response.EnsureSuccessStatusCode();
             }
         }
+
+        public async Task<HttpResponseMessage> SetExpiryDateAsync(string batchId, BatchExpiryModel batchExpiry)
```

Comment:
The client of this library should manage and control the cancellation token. Instead of using CancellationToken.None please make cancellation token a parameter (and also add to the interface definition too)

I realise this is inconsistent with the other interface methods but that's fine.

Reply:
Added as a mandatory pram for Cancellation token. Now, it is responsible of client to pass it.